### ISSUE TITLE
Package Examples of `fetchSCAN` Don't Work

### ISSUE THREAD

**Issue by colinbrust** [ncss-tech/soilDB#354]
When running the example given at [https://ncss-tech.github.io/AQP/soilDB/fetchSCAN-demo.html](https://ncss-tech.github.io/AQP/soilDB/fetchSCAN-demo.html), you get an error and are unable to download data using `fetchSCAN`. This also happens when using the example outlined in the function documentation:

``` r
library(soilDB)

packageVersion("soilDB")
#> [1] '2.8.2'
```

``` r

# result is a list
x <- soilDB::fetchSCAN(site.code=c(574), year=c(2016))
#> Error in `$<-.data.frame`(`*tmp*`, "Date", value = structure(numeric(0), class = "Date")) : 
#>   replacement has 0 rows, data has 368
```

``` r

# print a list of sensor types
names(x[-1])
#> NULL
```

<sup>Created on 2024-05-30 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

**Comment by colinbrust** [comment]
Never mind, I see this was closed by #350 and that these changes just aren't reflected in the CRAN package yet.

**Comment by brownag** [comment]
Thanks for your interest in soilDB and for reporting this. We will get a new version pushed up to CRAN pretty soon
